### PR TITLE
Removed newline characters.

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

File: v1.0/openapi3/headers/Content-Length.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 required: false
 schema:
   type: integer
-description: "The `Content-Length` header field indicates the anticipated size of the payload body. Only sent if there is a body.\n**Note:** The API supports a maximum size of 5242880 bytes (5 Megabytes)."
+description: "The `Content-Length` header field indicates the anticipated size of the payload body. Only sent if there is a body.**Note:** The API supports a maximum size of 5242880 bytes (5 Megabytes)."
```
Comment (stevenwjy) [inline v1.0/openapi3/headers/Content-Length.yaml:4]: Perhaps the description could be split into multiple lines like the others?

File: v1.0/openapi3/schemas/Name.yaml [hunk 2]
```diff
@@ -3,11 +3,11 @@ type: string
...
 description: |
   The API data type Name is a JSON String, restricted by a regular expression
-  to avoid characters which are generally not used in a name.\n
+  to avoid characters which are generally not used in a name.
   Regular Expression - The regular expression for restricting the
   Name type is \"^(?!\\s*$)[\\w .,'-]{1,128}$\". The restriction does not
   allow a string consisting of whitespace only, all Unicode characters are
   allowed, as well as the period (.) (apostrophe (‘), dash (-), comma (,)
-  and space characters ( ).\n\n\n**Note:** In some programming languages,
+  and space characters ( ).**Note:** In some programming languages,
```
Comment (stevenwjy) [inline v1.0/openapi3/schemas/Name.yaml:11]: Would it be better to put the note on a new line?

File: v1.0/openapi3/schemas/PartyIdType.yaml [hunk 3]
```diff
@@ -10,34 +10,34 @@ enum:
...
 description: |
-  Below are the allowed values for the enumeration.\n
+  Below are the allowed values for the enumeration.
   - MSISDN - An MSISDN (Mobile Station International Subscriber Directory
   Number, that is, the phone number) is used as reference to a participant.
   The MSISDN identifier should be in international format according to the
   [ITU-T E.164 standard](https://www.itu.int/rec/T-REC-E.164/en).
   Optionally, the MSISDN may be prefixed by a single plus sign, indicating the
-  international prefix.\n\n- EMAIL - An email is used as reference to a
+  international prefix.- EMAIL - An email is used as reference to a
```
Comment (stevenwjy) [inline v1.0/openapi3/schemas/PartyIdType.yaml:19]:
```suggestion
  international prefix.
  - EMAIL - An email is used as reference to a
```

File: v1.0/openapi3/headers/Content-Length.yaml (hunk 1 above)
Comment (stevenwjy) [reply v1.0/openapi3/headers/Content-Length.yaml:4]: Also, would it be better to separate the note into a new line?

File: v1.0/openapi3/info/index.yaml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 version: '1.0'
 title: 'Open API for FSP Interoperability (FSPIOP) (Implementation Friendly Version) Snippets'
-description: "Based on [API Definition version 1.0](https://github.com/mojaloop/mojaloop-specification/blob/develop/API%20Definition%20v1.0.pdf).\n**Note:** The API supports a maximum size of 65536 bytes (64 Kilobytes) in the HTTP header."
+description: "Based on [API Definition version 1.0](https://github.com/mojaloop/mojaloop-specification/blob/develop/API%20Definition%20v1.0.pdf).**Note:** The API supports a maximum size of 65536 bytes (64 Kilobytes) in the HTTP header."
```
Comment (stevenwjy) [inline v1.0/openapi3/info/index.yaml:3]: Same here, perhaps could be split into multiple lines? And would it be better to separate the note into a new line?

Comment (kleyow) [reply v1.0/openapi3/info/index.yaml]: done

File: v1.0/openapi3/schemas/PartyIdType.yaml (hunk 3 above)
Comment (kleyow) [reply v1.0/openapi3/schemas/PartyIdType.yaml]: done

File: v1.0/openapi3/schemas/Name.yaml (hunk 2 above)
Comment (kleyow) [reply v1.0/openapi3/schemas/Name.yaml]: done

File: v1.0/openapi3/headers/Content-Length.yaml (hunk 1 above)
Comment (kleyow) [reply v1.0/openapi3/headers/Content-Length.yaml]: done

**Pull request merged**